.github/workflows/test_app.yml

```diff
@@ -70,14 +70,6 @@ jobs:
           --health-retries 5
         env:
           POSTGRES_PASSWORD: postgres
-      redis:
```

Comment:
Why are you removing the Redis dependency?

Reply:
Because it was added at #12705 for this purpose and is not used anywhere else within the tests.